src/AstroBase.jl

```diff
@@ -1,5 +1,25 @@
 module AstroBase
 
+export tio_locator
+const J2000 = 2451545.0
+const DAYS_PER_CENTURY = 36525.0
+sec2rad(sec::Real) = deg2rad(sec / 3600)
```

Comment:
This one needs a docstring as well and while you're at it, you should also add the reverse operation.